File: Python/1038 - Snack.py
s = input()
l1 = s.split()
code, quantity = int(l1[0]),int(l1[1])
price_table = [4.00,4.50,5.00,2.00,1.50]
match code:
    case 1:
        price = (quantity*price_table[0])
        print("Total: R$ %.2f" % price)
    case 2:
        price = (quantity*price_table[1])
        print("Total: R$ %.2f" % price)
    case 3:
        price = (quantity*price_table[2])
        print("Total: R$ %.2f" % price)
    case 4:
        price = (quantity*price_table[3])
        print("Total: R$ %.2f" % price)
    case 5:
        price = (quantity*price_table[4])
        print("Total: R$ %.2f" % price)

File: C++/1038 - Snack.cpp
#include <iostream>
#include <vector>

double get_price(int item_code, int amount, std::vector<double> price_table)
{
    double price;
    price = price_table[item_code-1]*amount;
    return price;
}

int main()
{
    int item_code,amount;
    std::vector<double> price_table = {4.00, 4.50, 5.00, 2.00, 1.50};
    std::cin >> item_code >> amount;
    double price = get_price(item_code, amount, price_table);
    printf("Total: R$ %.2f\n",price);
    return 0;
}